target/linux/ath79/dts/qca9557_zyxel_nbg6616.dts

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	compatible = "zyxel,nbg6616", "qca,qca9557";
+	model = "ZyXEL NBG6616";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
```

Comment:
This has been moved to dtsi and can be dropped.

Reply:
done